core/kazoo_documents/src/kzd_limits.erl

```diff
@@ -459,6 +460,18 @@ allotments(Doc, Default) ->
 set_allotments(Doc, Allotments) ->
     kz_json:set_value(<<"allotments">>, Allotments, Doc).
 
+%%------------------------------------------------------------------------------
+%% @doc
+%% @end
+%%------------------------------------------------------------------------------
+-spec inbound_channels_per_did(doc()) -> kz_json:object().
+inbound_channels_per_did(Doc) ->
+    inbound_channels_per_did(Doc, kz_json:new()).
+
+-spec inbound_channels_per_did(doc(), Default) -> kz_json:object() | Default.
+inbound_channels_per_did(Doc, Default) ->
```

Comment:
Hey, 
This is looking good, I do have some questions about how these rules would be defined. 

The name inbound_channels_per_did seems to imply that these would be integer values and not a jobj which provides a set of rules for inbound did categorization and limitations. A more suitable name might be inbound_channels_per_did_rules since this is a jobj that contains a key which is either a regex or number and an integer value which is the limit applied to number(s) matching the key. 

Since this rule is slightly more complex than the existing limits settings, can you please document this with example rules and the expected behavior for how the rules are enforced? That way people wanting to use this feature have an easy way to see what they can define and how those definitions would impact their limitations. 

Thank you!

Reply:
@swysor Thanks for a comment. Agreed. Will do in couple of days

Reply:
Sounds good @onnet, thank you!

Reply:
@swysor Hi! addresed

Reply:
Thanks, I made a small edit (as a suggestion) for the documentation. Once this is addressed I can merge this. 

Also can you please cherry-pick this back to 4.3 and do a PR against that branch as well?

Thank you!

Reply:
@swysor Hi 
PRed against 4.3 https://github.com/2600hz/kazoo/pull/5793